base/src/main/kotlin/net/corda/v5/base/types/MemberX500Name.kt

```diff
@@ -76,9 +169,13 @@ class MemberX500Name(
         }
 
         /**
-         * Creates an instance of [MemberX500Name] from specified [X500Principal]
+         * Creates an instance of [MemberX500Name] from specified [X500Principal].
+         *
+         * @param principal The X500 principal used for building [MemberX500Name].
          *
          * @throws [IllegalArgumentException] if required attributes are missing, constrains are not satisfied.
+         *
+         * @return [MemberX500Name] based on param.
```

Comment:
Would it be better to link the actual param here, i.e. [principal], instead of "param"? Same for other methods in this class